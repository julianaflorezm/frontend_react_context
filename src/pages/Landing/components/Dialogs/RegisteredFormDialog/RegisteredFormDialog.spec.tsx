import { render } from '@testing-library/react';
import { RegisteredFormDialogProps, RegisteredFormDialog } from './index';

describe('RegisteredFormDialog Test', () => {
  let props: RegisteredFormDialogProps;

  beforeEach(() => {
    props = {
      open: true,
      onClose: jest.fn(),
      title: 'regÃ­strese',
      description: 'description',
    };
  });

  it('should match snapshot', () => {
    const { container } = render(<RegisteredFormDialog {...props} />);
    expect(container).toMatchSnapshot();
  });
});
