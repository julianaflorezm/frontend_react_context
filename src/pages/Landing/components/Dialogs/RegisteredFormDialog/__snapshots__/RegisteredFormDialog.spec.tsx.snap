// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegisteredFormDialog Test should match snapshot 1`] = `
<div
  aria-hidden="true"
/>
`;
